.github/workflows/pages.yml

```diff
@@ -0,0 +1,61 @@
+# Simple workflow for deploying static content to GitHub Pages
+name: Deploy static content to Pages
+
+on:
+  # Runs on pushes targeting the default branch
+  push:
+    branches: ["docs-test"]
```

Comment:
Do you want to keep this specific branch name as part of the trigger?

Reply:
Good catch, I had forgotten to push the last set of changes. Fixed.